platforms/chibios/drivers/i2c_master.c

```diff
@@ -27,8 +27,80 @@
 #include "quantum.h"
 #include "i2c_master.h"
 #include <string.h>
+#include <ch.h>
 #include <hal.h>
 
+#ifdef I2C1_BANK
+#    define I2C1_SCL_BANK I2C1_BANK
```

Comment:
If this is moved out of the header file, something like this will start failing: See keyboards/matrix/m20add/m20add.c line 67